procurement_plan/wizard/wiz_load_purchase_from_plan.py

```diff
@@ -0,0 +1,180 @@
+    product_id = fields.Many2one("product.product", string="Product")
+    factor = fields.Float(string="Factor", default=1)
+
+    @api.onchange('purchase_id')
+    def purchase_onchange(self):
+        if self.purchase_id:
+            self.partner_id = self.purchase_id.partner_id.id
+            self.date_from = self.purchase_id.date_order
+            self.date_to = self.purchase_id.date_order
+
+    @api.multi
+    def load_purchases(self):
+        self.ensure_one()
+        procurement_obj = self.env['procurement.order']
+        plan = self.env['procurement.plan'].browse(
+            self.env.context['active_id'])
+        purchase_lines = self.get_purchase_lines()
+        date_list = self.get_date_list(plan)
+        date_start = fields.Date.from_string(self.date_from)
+        date_end = fields.Date.from_string(self.date_to)
+        month_count = ((date_end.year - date_start.year) * 12 +
+                       date_end.month - date_start.month + 1)
+        result = self.match_purchases(purchase_lines, self.factor)
+        for date in date_list:
+            for partner in result.keys():
+                for product in result[partner].keys():
+                    prod_vals = result[partner][product]
+                    vals = {'name': plan.name,
+                            'origin': plan.sequence,
+                            'product_id': product,
+                            'plan': plan.id,
+                            'main_project_id': plan.project_id.id,
+                            'product_qty': prod_vals['qty'] / month_count,
+                            'warehouse_id': self.warehouse_id.id,
+                            'location_id':  self.warehouse_id.lot_stock_id.id,
+                            'date_planned': date
+                            }
+                    vals.update(
+                        procurement_obj.onchange_product_id(product)['value'])
+                    procurement_obj.create(vals)
+        return True
+
+    @api.multi
+    def get_purchase_lines(self):
+        purchase_line_obj = self.env['purchase.order.line']
+        purchase_obj = self.env['purchase.order']
+        product_obj = self.env['product.product']
+        self.ensure_one()
+        purchases = []
+        if self.purchase_id:
+            purchases = self.purchase_id
+        else:
+            purchase_domain = [('date_order', '>=', self.date_from),
+                               ('date_order', '<=', self.date_to)]
+            if self.partner_id:
+                purchase_domain += [('partner_id', '=', self.partner_id.id)]
+            purchases = purchase_obj.search(purchase_domain)
+        purchase_line_domain = [('order_id', 'in', purchases.ids)]
+        if self.product_id:
+            purchase_line_domain += [('product_id', '=', self.product_id.id)]
+        elif self.product_tmpl_id:
+            purchase_line_domain += [('product_tmpl_id', '=',
```

Comment:
Y aún así. Creo que el campo se llamaba product_template. En cualquier caso, hay que quitar esto y convertirla a 'product_id', 'in', los productos de esa plantilla